### ISSUE TITLE
A11Y_Visual Studio Code Services_Docs User Guide_IntelliSense_ScreenReader:The decorative graphics in column one are announced on navigating using table keys 

### ISSUE THREAD

**Issue by v-jagansai** [microsoft/vscode-docs#5568]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win11;#FTP;#DesktopWeb;#A11ySev3;#WCAG1.3.1;#NVDA;#JAWS;#STP;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win11
Chrome Version: Version 103.0.5060.134 (Official Build) (64-bit)
NVDA Version:2021.3.5,JAWS

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Navigate to "Docs" link.
4. From left navigation, select "User Guide".
5. Navigate to "IntelliSense" link.
6. Start Screen Reader .
7. Tab to table presented in the screen .
8. Observe that the Screen reader is announcing the graphic images as column one.

## Actual:
The decorative graphics in column one are announced on navigating using table keys .

Same issue is seen with Chrome+JAWS.

## Expected:
On navigating to the Graphics in column one using table keys they should not be announce the table should be read as 21 rows and 2 columns. Screen reader navigation should not move to the column that contains graphic images.
 
## User Impact:
Screen reader users will get decorative image information.

## Attachment:
![Presentation8](https://user-images.githubusercontent.com/109065315/184312485-58c367f0-79ef-401b-b98e-27b2a36058fe.gif)
![graph](https://user-images.githubusercontent.com/109065315/184351543-c64e9a30-4fb7-4a67-bd97-03d81457a56a.gif)

**Comment by gregvanl** [comment]
Not sure I understand but the icons are in column one are part of the table.
Other issues with that table were fixed with https://github.com/microsoft/vscode-docs/commit/9360da4c7861288e791ab18275fbddb5da8f494d
